src/Transport.java

```diff
@@ -0,0 +1,29 @@
+public class Transport {
+    public String modelName;
+    public int wheelsCount;
+
+    public Transport(String modelName, int wheelsCount) {
+        this.modelName = modelName;
+        this.wheelsCount = wheelsCount;
+    }
+
+    public String getModelName() {
+        return modelName;
+    }
+
+    public void setModelName(String modelName) {
+        this.modelName = modelName;
+    }
+
+    public int getWheelsCount() {
+        return wheelsCount;
+    }
+
```

Comment:
Так как мы хотим реализовать интерфейс Service, то нужно реализовать его метод:

```suggestion
    @Override
    public void check() {
        //тут добавить реализацию(код), который проверят покрышки
    }
```